attendees_and_learners.rst

```diff
@@ -1,4 +1,4 @@
-####################
+﻿####################
```

Comment:
Hi Amy. It looks like you're using a machine with a Windows line-ending character.

https://help.github.com/articles/dealing-with-line-endings/ has more info - don't hesitate to ask me or anyone else for help fixing it.